tests/nested/manual/core20-boot-stable-kernel/task.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Check snapd built from current branch boots with the stable kernel
+
+systems: [ubuntu-20.04-*]
+
+environment:
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+    NESTED_USE_CLOUD_INIT: true
+    NESTED_ENABLE_SECURE_BOOT: true
+    NESTED_ENABLE_TPM: true
+
+    NESTED_KERNEL_CHANNEL: stable
+    NESTED_GADGET_CHANNEL: stable
+    NESTED_BASE_CHANNEL: stable
+    NESTED_SNAPD_CHANNEL: edge
```

Comment:
along with this, if we don't already have this in one of the other spread tests, it would be good to have the following variants:

- [x] edge + build_from_branch snapd + stable everything else (this test)
- [ ] edge + build_from_branch snapd + beta kernel + stable everything else
- [ ] edge + build_from_branch snapd + beta gadget + stable everything else
- [ ] edge + build_from_branch snapd + beta base (core20) + stable everything else

Reply:
fine for a followup too